pkg/router/template/router.go

```diff
@@ -131,6 +132,8 @@ type templateData struct {
 	State map[string](ServiceAliasConfig)
 	// the service lookup
 	ServiceUnits map[string]ServiceUnit
+	// number of service endpoints
+	ServiceEndpoints map[string]int
```

Comment:
Don't you already have this from the service state?

Reply:
+1 looks like we can use len(serviceUnit.EndpointTable)?

Reply:
I will change this. It was just a convenience.